tools/purge_orphans.rb

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env ruby
+require "optimist"
+
+VALID_MODES = %w[count purge].freeze
+opts = Optimist.options do
+  banner "Purge orphaned rows from several tables.\n\nUsage: ruby #{$PROGRAM_NAME} [options]\n\nOptions:\n\t"
+  opt :mode, "Mode (#{VALID_MODES.join(", ")})", :default => "count", :permitted => VALID_MODES
+end
+
+purge_mode = opts[:mode].to_sym
+
+# now load rails
+require File.expand_path('../config/environment', __dir__)
+
+def purge_by_orphaned(klass, fk, window, purge_mode = false)
```

Comment:
I don't think false is a valid value here?  Think you can just remove this default value.

Reply:
done